device-types/Cisco/WS-C2950G-24-EI.yaml

```diff
@@ -0,0 +1,67 @@
+---
+manufacturer: Cisco
+model: Catalyst 2950G-24-EI
+part_number: WS-C2950G-24-EI
+slug: cisco-ws-c2950g-24-ei
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: con 0
+    type: rj-45
+power-ports:
+  - name: POWER
+    type: iec-60320-c14
+    maximum_draw: 45
```

Comment:
Dont these have a DC secondary power supply port? Specifically for a 14 pin DC terminal? If so, please add a power port with `dc-terminal` type